pom.xml

```diff
@@ -21,34 +21,58 @@
   </licenses>
 
   <properties>
-    <java.version>11</java.version>
+    <java.version>17</java.version>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <folio-spring-base.version>5.0.2</folio-spring-base.version>
-    <edge-common-spring.version>2.0.2</edge-common-spring.version>
-    <openapi-generator.version>5.4.0</openapi-generator.version>
+    <folio-spring-base.version>6.0.0-SNAPSHOT</folio-spring-base.version>
+    <edge-common-spring.version>3.0.0-SNAPSHOT</edge-common-spring.version>
```

Comment:
It should be merged after this pr https://github.com/folio-org/edge-common-spring/pull/28